README.md

```diff
@@ -8,13 +8,13 @@
 
 [![dockeri.co](https://dockeri.co/image/zixia/simple-mail-forwarder)](https://hub.docker.com/r/zixia/simple-mail-forwarder/)
 
-Simplest and Smallest Email Forward Service based on Docker.
+Simplest and Smallest Email Forward Service for Docker.
 
-1. Config by [**one-line**](#environment-variable-and-default)
+<!-- @Deprecated 1. Config by [**one-line**](#environment-variable-and-default) -->
 1. Run as [**docker start**](#quick-start-tldr)
 1. Image Size [**10MB**](https://hub.docker.com/r/zixia/simple-mail-forwarder/tags/)
 
-View on Github - https://github.com/huan/docker-simple-mail-forwarder
+[View on Docker](https://hub.docker.com/r/zixia/simple-mail-forwarder)
```

Comment:
Thanks for adding the docker hub link.

However, please keep the GitHub link when you adding the DockerHub link.

Those two should all be displayed in our README.

Reply:
I understand!